pkg/volume/gce_pd/gce_util.go

```diff
@@ -58,6 +66,12 @@ func (util *GCEDiskUtil) AttachAndMountDisk(pd *gcePersistentDisk, globalPDPath
 	devicePath := ""
 	// Wait for the disk device to be created
 	for {
+		sdAfter, err := filepath.Glob("/dev/sd*")
+		if err != nil {
+			glog.Errorf("Error filepath.Glob(\"/dev/sd*\"): %v\r\n", err)
+		}
+		udevadmChangeToNewDrives(sdBefore, sdAfter)
+
```

Comment:
@dchen1107 @lavalamp I'm being a bit lazy here, but what is the preferred way of doing these sorts of retries inside kubelet?